build.sbt

```diff
@@ -16,6 +16,7 @@
 lazy val scala211Version = "2.11.12"
 lazy val scala212Version = "2.12.10"
 lazy val sparkVersion = "2.4.0"
+lazy val nettyVersion = "4.1.30.Final"
```

Comment:
Should we use the latest Netty `4.1.50.Final` version: https://search.maven.org/artifact/io.netty/netty-all ?

Also, you should keep in sync Netty and gRPC versions: https://github.com/grpc/grpc-java/blob/master/SECURITY.md#netty

Or you can avoid this sync issue by using `grpc-netty-shaded` gRPC artifact instead of Netty artifacts.

Reply:
We are using the shaded version. I've upgraded to `4.1.50.Final`

Reply:
In this case you should drop all `io.netty:` dependencies because you are using `grpc-netty-shaded` dependency that has an appropriate version for the gRPC version.

Reply:
I can't - Arrow adds dependency on netty in version `4.1.27.Final` so we need to explicitly upgrade those.

Reply:
Interesting, in this case we should not use `grpc-netty-shaded` for gRPC, because BQ connector will have 2 Netty artifacts - one shaded for gRPC and one not shaded for Arrow? In this case, I think, both gRPC and Arrow should use the same unshaded Netty.